src/prefect/utilities/executors.py

```diff
@@ -261,11 +261,11 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
             except RecursiveCall as exc:
                 try:
                     call_func = getattr(exc.func, "__wrapped_func__")
-                except AttributeError:
+                except AttributeError as exc:
                     raise RecursionError(
                         "function has not been wrapped to provide tail recursion (func={})".format(
                             exc.func
-                        )
+                        ) from exc
```

Comment:
I think this needs to be moved down one parenthesis

Reply:
Ahhh! I see it. Let me fix that quick.